pkg/tagger/remote/tagger.go

```diff
@@ -70,7 +68,6 @@ func NewTagger() *Tagger {
 // Init initializes the connection to the remote tagger and starts watching for
 // events.
 func (t *Tagger) Init(ctx context.Context) error {
-	t.health = health.RegisterLiveness("tagger")
```

Comment:
Do we not want any health reporting anymore for the tagger?

Reply:
From the second commit in this PR:

> First of all, this health check has never worked, as in, it has never
consistently passed. We just never noticed because the other agents
(process, trace, ...) don't actually have health checks configured.
>
> Second of all, there's no obvious way to make it work with a gRPC
stream, since it is designed to block until there's a new message.
Having this receive on a separate goroutine is pointless as well, since
it's the very thing we want to monitor for "stuck-ness". So I'm electing
to have this removed entirely.